static/js/settings_account.js

```diff
@@ -64,6 +64,10 @@ function update_user_custom_profile_fields(fields, method) {
 }
 
 exports.append_custom_profile_fields = function (element_id, user_id) {
+    var person = people.get_person_from_user_id(user_id);
+    if (person.is_bot) {
+        return null;
```

Comment:
This should be just `return;`, since this function doesn't return anything normally.

Reply:
Right.